cmd/tailscale/cli/cli_test.go

```diff
@@ -632,7 +631,7 @@ func TestPrefsFromUpArgs(t *testing.T) {
 			st: &ipnstate.Status{
 				TailscaleIPs: []netaddr.IP{netaddr.MustParseIP("100.105.106.107")},
 			},
-			wantErr: `cannot use 100.105.106.107 as the exit node as it is a local IP address to this machine, did you mean --advertise-exit-node?`,
+			wantErr: `100.105.106.107: cannot use this address as an exit node as it is a local IP address to this machine; did you mean --advertise-exit-node?`,
```

Comment:
I kinda liked the old word order more ("the" to "an" is fine, and semicolon is better, though)

Reply:
This is a consequence of the fact that the patch splits some of that error message by moving some of that code into `prefs.go` while leaving some of it in `up.go`. (Some parts of that message seemed more appropriate for just the command line use case.)

I'll go over this and reconsider.

Reply:
OK I redid this in a way that preserves the ordering in this message.